extensions/amp-video/0.1/amp-video.js

```diff
@@ -192,6 +192,11 @@ class AmpVideo extends AMP.BaseElement {
     const {element} = this;
 
     this.configure_();
+    this.intersectionObserver_ = new IntersectionObserver(entries => {
+      const visible = entries[entries.length-1].isIntersecting;
+      this.element.dispatchCustomEvent(VideoEvents.VISIBILITY, {visible}); 
```

Comment:
I know this was an old implementation, but can we confirm whether we should be sending this event for a non-loaded video? /cc @alanorozco